### PR TITLE
feat(view-only): show badge on sandboxes and repos

### PR DESCRIPTION
**Pull request by alexnm**
Quite some changes, but mostly updating the `Badge` component to reflect the prism version: 20px height, 12px font-size and same API

Screenshot 2022-11-16 at 19 02 33
Screenshot 2022-11-16 at 19 02 44
Screenshot 2022-11-16 at 19 02 49
Screenshot 2022-11-16 at 19 03 01

Still todo:
- [ ] remove some context menu items for view only sandboxes and repositories
- [ ] add a stripe in the private repo (branch list) with a specific reference to the private repo on a free tier

### CODE REVIEW

File: packages/app/src/app/components/CreateSandbox/CreateSandbox.tsx [hunk 1]
```diff
@@ -335,10 +334,7 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
                     }}
                     stopId="cloud-templates"
                   >
-                    <Stack gap={2}>
-                      <span>Cloud templates</span>
-                      <Badge>Beta</Badge>
-                    </Stack>
```
Comment (alexnm) [inline packages/app/src/app/components/CreateSandbox/CreateSandbox.tsx:341]: Badge doesn't play well with the line height here. If it's important, I'll add a ticket later to add it back properly, but we will soon remove the beta anyway, so..

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7161/commits/1a753693493d52812746edd99641516be525a04c is at https://pr7161.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx [hunk 2]
```diff
@@ -147,6 +149,7 @@ const GenericSandbox = ({ isScrolling, item, page }: GenericSandboxProps) => {
 
   const selected = selectedIds.includes(sandbox.id);
   const isDragging = isAnythingDragging && selected;
+  const isViewOnly = !hasActiveSubscription && sandbox.privacy !== 0;
```
Comment (olarclara) [inline packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx:152]: Privacy being an integer: 🥲

**Review by olarclara — APPROVED** [review]

File: packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx (hunk 2 above)
Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx:152]: Raised this with Julien, it's the same on the backend unfortunately. We can define an enum in the FE maybe to make it friendlier

Comment (tristandubbeld) [reply packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx:152]: @julien-leclercq wants to change this to something else too :)

**Review by tristandubbeld — APPROVED** [review]
Cleans up nicely 🧹

**Pull request merged**